src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.html

```diff
@@ -15,10 +15,10 @@
        href="{{ docsUrl }}"
        target="_blank"
        i18n>documentation</a>
-    <button ngbDropdownItem
-            routerLink="/api-docs"
-            target="_blank"
-            i18n>API</button>
+    <a ngbDropdownItem
+       routerLink="/api-docs"
+       target="_blank"
+       i18n>API</a>
```

Comment:
I think we should be adding the external Icon as well.


```suggestion
    <a ngbDropdownItem
       routerLink="/api-docs"
       target="_blank"
       i18n>API
         <i class="fa fa-external-link"></i>
       </a>
```

Reply:
Added it both for documentation and API links